index.html

```diff
@@ -1569,10 +1569,24 @@
           Messaging between the local and remote playback devices
         </h3>
         <p>
-          This spec will not mandate communication protocols between the
-          <a>local playback device</a> and the <a>remote playback device</a>,
-          but it should set some guarantees of message confidentiality and
-          authenticity between them.
+          This specification will not mandate communication protocols between
+          the <a>local playback device</a> and the <a>remote playback
+          device</a>, but it should set some guarantees of message
```

Comment:
It's not clear what "it" means here - the specification, or one of the devices?

Reply:
I went with "user agent".